system/indy-sdk-tests/test_libindy_acceptance_v2.py

```diff
@@ -0,0 +1,82 @@
+import docker
+import testinfra
+import time
+from system.docker_setup import NETWORK_NAME
+
+
+def test_libindy():
+    client = docker.from_env()
+    client_name = 'libindy_acceptance_client'
+    pool_name = 'libindy_acceptance_pool'
+    sdk_ver = '1.12.0'
+
+    try:
+        client.containers.get(client_name).remove(force=True)
+    except docker.errors.NotFound:
+        pass
+
+    try:
+        client.containers.get(pool_name).remove(force=True)
+    except docker.errors.NotFound:
+        pass
+
+    # pool setup
+    pool_image, output = client.images.build(
+        path='.', dockerfile='_libindy_acceptance_pool.dockerfile', tag=pool_name
+    )  # use `buildargs` (dict): A dictionary of build arguments
+    print('\nBuild logs:')
+    for line in output:
+        print(line)
+    # use `environment` (dict or list): Environment variables to set inside the container
+    pool_container = client.containers.run(pool_image, name=pool_name, network='host', detach=True, tty=True)
+    pool_container.start()
+
+    # client setup
+    client_image, output = client.images.build(
+        path='.', dockerfile='_libindy_acceptance_client.dockerfile', tag=client_name
+    )  # use `buildargs` (dict): A dictionary of build arguments
+    print('\nBuild logs:')
+    for line in output:
+        print(line)
+    # use `environment` (dict or list): Environment variables to set inside the container
+    client_container = client.containers.run(client_image, name=client_name, network='host', detach=True, tty=True)
```

Comment:
Is it really necessary to attach container to host network? Why tests couldn't run in their own private net?